src/components/ReviewConfirm/Summary.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <div class="summary-container">
-    <SummaryDefineCompany />
+    <SummaryDefineCompany/>
     <ListPeopleAndRoles :personList="orgPersonList" :isSummary="true" :showErrorSummary="!step2Valid" />
     <ListShareClass :shareClasses="shareClasses" :isSummary="true" :showErrorSummary="!step3Valid"/>
+    <AgreementType :isSummary="true" :showErrorSummary="!step4Valid"/>
```

Comment:
I *think* you can just specify the attribute when it's "true", eg,
`<AgreementType isSummary :showErrorSummary="!step4Valid" />`

If so, it would be a bit cleaner for all 3 summaries here.

Reply:
It dint work that way. It shows the component in edit mode. So  I am leaving it as it is now.

Reply:
Darn. It works that way for Vuetify components.